osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-configuration.xsd

```diff
@@ -192,6 +225,39 @@
     </xsd:complexType>
   </xsd:element>
 
+  <xsd:element name="GetFirmwareVersionGasRequest">
+    <xsd:complexType>
+      <xsd:sequence>
+        <xsd:element name="DeviceIdentification" type="common:Identification"/>
```

Comment:
I would expect `common:DeviceIdentification` to be a more specific and appropriate type to use here, any reason to go for the broader `common:Identification`?

Reply:
I think I 'borrowed' this from ActualMeterReadsGas. I'll have a look at that.